packages/templates/nextjs-marketing-demo/src/getExperience.ts

```diff
@@ -8,6 +8,8 @@ const environment = process.env.NEXT_PUBLIC_CTFL_ENVIRONMENT!;
 const experienceTypeId = process.env.NEXT_PUBLIC_CTFL_EXPERIENCE_TYPE!;
 const domain = process.env.NEXT_PUBLIC_CTFL_DOMAIN || 'contentful.com';
 
+console.log('accessToken', accessToken, 'prevAccessToken', prevAccessToken);
```

Comment:
Is this intentional? Could be better ways to check if the token exists or not (ie check against undefined and throw errors) if that's the intent of the console log

Reply:
nope, forgot to remove it, good catch